app/Http/Requests/Sharing/AddSharingRequest.php

```diff
@@ -39,7 +39,7 @@ class AddSharingRequest extends BaseApiRequest implements HasAlbumIds, HasUserId
 	 */
 	public function authorize(): bool
 	{
-		return Gate::check(AlbumPolicy::CAN_SHARE_ID, [AbstractAlbum::class, $this->albumIds]);
+		return Gate::check(AlbumPolicy::CAN_SHARE_ID, [AbstractAlbum::class, $this->albumIds()]);
```

Comment:
`$this->albumIds() ` 

is this intended?

Reply:
yes. It is also provided in the trait :)